packages/components/src/font-size-picker/index.js

```diff
@@ -53,17 +53,15 @@ function getSelectOptions( optionsArray, disableCustomFontSizes ) {
 	} ) );
 }
 
-function FontSizePicker(
-	{
-		fallbackFontSize,
-		fontSizes = [],
-		disableCustomFontSizes = false,
-		onChange,
-		value,
-		withSlider = false,
-	},
-	ref
-) {
+function FontSizePicker( {
+	fallbackFontSize,
+	fontSizes = [],
+	disableCustomFontSizes = false,
+	onChange,
+	value,
+	withSlider = false,
+	ref,
+} ) {
```

Comment:
Why this change to move `ref` into the props themselves? I thought forwarded refs were always passed as a second parameter to the functional component?

Reply:
Thank you @saramarcondes, it was fixed 👍